backend/printing/views.py

```diff
@@ -0,0 +1,57 @@
+from pyqrcode import QRCode
+import io
+from django.http import HttpResponse
+from django.utils.datastructures import MultiValueDictKeyError
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status, generics
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+
+@api_view(["GET"])
+def CreatePrintable(request):
```

Comment:
This could be moved to django-rest-framework class view with overriden get method.